nixos/modules/services/networking/tailscale.nix

```
@@ -31,6 +38,13 @@ in {
 
   config = mkIf cfg.enable {
     environment.systemPackages = [ cfg.package ]; # for the CLI
+    networking.firewall = mkIf cfg.openFirewall {
+      trustedInterfaces = if cfg.interfaceName != "userspace-networking" then
```

Let us have one less `if .. then .. else ..`:
```suggestion
      trustedInterfaces = optional (cfg.interfaceName != "userspace-networking") cfg.interfaceName
```